services/slam/builtin/builtin_test.go

```diff
@@ -772,10 +781,10 @@ func TestORBSLAMDataProcess(t *testing.T) {
 
 		n := 5
 		// Note: timePadding is required to allow the sub processes to be fully completed during test
-		time.Sleep(time.Millisecond * time.Duration((n)*(validDataRateMS+timePadding)))
+		time.Sleep(time.Millisecond * time.Duration((n)*(validDataRateMS+timePadding*2)))
```

Comment:
I'm surprised that 40ms of padding is needed here, since returning the image from the injected camera and writing it to a file should be fast. Also it's possible that we'll get 6 images here, since the sleep now lasts for 5*(200+40)=1200ms. What error do you get if you only use 20ms of padding? And if you need to have 40ms of padding, do you want to change the check at the end of the test to require the number of images is greater than or equal to 5, just in case we get 6 images?

Reply:
I get an error that only 4 images were saved - while 5 images are expected. Setting it to 40 ms allows it to save 5 images. Setting it to 38 would still only return 4 images. Thus, I think the risk of it returning more than 5 images is small, but requiring `>= 5` images makes sense.